src/controllers/userController.js

```diff
@@ -16,45 +15,20 @@ class UserController {
    * @return {json} Returns json object
    * @memberof UserController
    */
-  static async login(req, res, next) {
+  static async login(req, res) {
     try {
-      const { email, password } = req.body;
-      const user = await Users.findOne({
-        where: {
-          email
-        }
-      });
-      if (!user) {
-        return res.status(400).json({
-          success: false,
-          message: 'Invalid email or password',
-        });
+      const user = await UserService.login(req);
```

Comment:
Is there a reason why we are creating a UserService abstraction here?. I thought this PR was to setup middleware with JWT

Reply:
Yes, his code was a bit weird to abstract away and use the Auth.sign method inside and I thought of rewriting it in what I felt was more modular. I could revert it if you want but I honestly don't know how to use the Auth.sign method in what he has.